### ISSUE TITLE
Introduce a single configuration file for all things Serenity/JS

### ISSUE THREAD

**Issue by jan-molak** [serenity-js/serenity-js#594]
It might be useful to introduce a way to configure all Serenity/JS modules in one place.

The config file could look as follows:
```
module.exports = {
    'core': {
        actors: new Actors(),
        crew: [
            ArtifactArchiver.storingArtifactsAt('./target/site/serenity'),
            new SerenityBDDReporter(),
            ConsoleReporter.forDarkTerminals(),
        ],
    },
    'serenity-bdd': {
        repository: 'https://artifactory.superevilmegacorp.com',
        ignoreSSL: true,
        auth: {
            username: '...',
            password: '...',
        },
        cacheDir: 'node_modules/@serenity-js/cache',
        reportDir: 'target/site/serenity',
        featuresDir: 'features',
        artifact: 'net.serenity-bdd:serenity-cli:jar:all:2.1.10',
        sourceDir: 'target/site/serenity',
        issueTrackerUrl: '',
        jiraProject: '',
        jiraUrl: '',
        project: '',
        log: 'warn'
    },
    'protractor': {
        runner: 'jasmine',
        timeouts: {
            pageOpen: 5000,
            wait: 1000,
        }
    },
    'local-server': {
        port: 8080
    }
}
```

**Comment by Ijee** [comment]
This is something I'd seriously like to see as adding more into the protractor config would just clutter it even more.

Would you try to patch the protractor config with the necessary information that are still required for it to work during runtime or would we still need to put at least some things in the protractor config file?

This would probably also advance #493  at least in some parts.

**Comment by jan-molak** [comment]
> Would you try to patch the protractor config with the necessary information that are still required for it to work during runtime or would we still need to put at least some things in the protractor config file?

That's a good question, I'm not sure yet. It _feels_ cleaner to keep protractor config in `protractor.conf.js` and Serenity/JS-specific stuff in a separate file. Plus, Protractor is still responsible for spawning and managing the browser so we can't extract say `capabilities` or `selenium` config from `protractor.conf.js` and put it in S/JS config file.

I think the same logic would apply to the integration with WebDriverIO, Cypress and any other test runner. 

What do you think?

**Comment by Ijee** [comment]
> That's a good question, I'm not sure yet. It feels cleaner to keep protractor config in protractor.conf.js and Serenity/JS-specific stuff in a separate file.

True. It was just something I threw in without thinking about it a little more.
